pyt/__main__.py

```diff
@@ -30,7 +30,25 @@
 )
 
 
-def main(command_line_args=sys.argv[1:]):  # noqa: C901
+def discover_files(targets, excluded_files, recursive=False):
+    included_files = list()
+    excluded_list = excluded_files.split(",")
+
+    for target in targets:
+        if os.path.isdir(target):
+            if recursive:
```

Comment:
So having `if recursive:` here it will make it so that if you don't have `-r` then you won't search directories.

You can change it to:

```python
def discover_files(targets, excluded_files, recursive=False):
    included_files = list()
    excluded_list = excluded_files.split(",")

    for target in targets:
        if os.path.isdir(target):
            for root, dirs, files in os.walk(target):
                for f in files:
                    fullpath = os.path.join(root, f)
                    if os.path.splitext(fullpath)[1] == '.py' and fullpath.split("/")[-1] not in excluded_list:
                        included_files.append(fullpath)
                if not recursive:
                    break
        else:
            if target not in excluded_list:
                included_files.append(target)
    return included_files
```